### PR TITLE
Add videocall plugin

### PR DESCRIPTION
**Pull request by thrfox**
thank you for this janus-api.
my project need to use videocall plugin. so i tried to code videocall api

### CODE REVIEW

File: src/plugin/VideoCallPlugin.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/* eslint-disable no-console */
```
Comment (tgabi333) [inline src/plugin/VideoCallPlugin.js:1]: could you please remove this line

File: src/plugin/VideoCallPlugin.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+   *  - "fallback" : <How much time (in us, default 250000) without receiving packets will make us drop to the substream below>
+   */
+  configure (set) {
+    const request = { request: 'set' }
+    const body = Object.assign({}, request, set)
+    return this.transaction('message', { body }, 'event')
+  }
+
+  hangup () {
+    // super.hangup()
```
Comment (tgabi333) [inline src/plugin/VideoCallPlugin.js:70]: is there any reason not to call super.hangup()?

File: src/plugin/VideoCallPlugin.js (hunk 1 above)
Comment (thrfox) [reply src/plugin/VideoCallPlugin.js]: done

File: src/plugin/VideoCallPlugin.js (hunk 2 above)
Comment (thrfox) [reply src/plugin/VideoCallPlugin.js]: i update it. It should be emit 'hangup' whether out side or the other side

**Comment by tgabi333** [comment]
@thrfox thanks, merged, released as 6.2.0

**Pull request merged**